blocks/test/fixtures/core__more.html

```diff
@@ -0,0 +1 @@
+<!-- wp:core/more {"text": "Continue reading", "noTeaser": true} /-->
```

Comment:
How did you generate this?  Since it won't do anything in WP core - and since it's different from the default serialization - I think it should use the `<!--more-->` format.